docs/advanced/document-store/index.md

```diff
@@ -564,6 +564,43 @@ The solution is simple: use {ref}`column-selector<bulk-access>`:
 da[0, 'text'] = 'hello'
 ```
 
+### Offset2id is slow
```

Comment:
I do not like tbis title, simply put Disable List-Like behavior or something like that. Or improve performance